app/grandchallenge/challenges/models.py

```diff
@@ -664,6 +665,65 @@ def create_default_pages(self):
             permission_level=Page.ALL,
         )
 
+    def create_default_onboarding_tasks(self):
+        OnboardingTask.objects.create(
+            challenge=self,
+            title="Create Phases",
+            description="Define, name and create the different phases of the challenge.",
+            responsible_party=OnboardingTask.ResponsiblePartyChoices.CHALLENGE_ORGANIZERS,
+            deadline=self.created + timedelta(weeks=1),
+        )
+        OnboardingTask.objects.create(
+            challenge=self,
+            title="Define Inputs and Outputs",
+            description="To support, communicate the required input and output data formats for participant's algorithms.",
+            responsible_party=OnboardingTask.ResponsiblePartyChoices.CHALLENGE_ORGANIZERS,
+            deadline=self.created + timedelta(weeks=3),
+        )
+        OnboardingTask.objects.create(
+            challenge=self,
+            title="Create Archives",
+            description="Create an archive per algorithm-type phase for the challenge.",
+            responsible_party=OnboardingTask.ResponsiblePartyChoices.SUPPORT,
+            deadline=self.created + timedelta(weeks=3),
+        )
+        OnboardingTask.objects.create(
+            challenge=self,
+            title="Upload Data",
+            description="Prepare and upload necessary datasets to archives, contact support if no archives are available.",
```

Comment:
Do the challenge organisers see that the archives need to be created by support first?

Reply:
They do not see the support task ticket for this.